.github/workflows/ci.yml

```diff
@@ -151,47 +151,36 @@ jobs:
       run: |
         codecov -n "GitHub Actions - ${{ matrix.task.name}} - ${{ matrix.os.name }} ${{ matrix.python.name }}"
 
-  check:
-    name: ${{ matrix.task.name}} - ${{ matrix.python.name }}
+  lint:
+    name: Linters
     runs-on: ubuntu-latest
-    needs:
-      - build
-    strategy:
-      fail-fast: false
-      matrix:
-        python:
-          # Using second most recent minor release for whatever little
-          # increase in stability over using the latest minor.
-          - name: CPython 3.9
-            tox: py39
-            action: 3.9
-        task:
-          - name: Check Newsfragment
-            tox: check-newsfragment
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
 
     steps:
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
-    - name: Download package files
-      uses: actions/download-artifact@v3
-      with:
-        name: dist
-        path: dist/
-
-    - name: Set up ${{ matrix.python.name }}
+    - name: Set up python
       uses: actions/setup-python@v3
       with:
-        python-version: ${{ matrix.python.action }}
+        python-version: "3.10"
```

Comment:
These days I tend to keep my checks one version behind (3.9 not 3.10).  They are there to check the code, not see if the tools work with the latest Python.  But definitely not a big deal, just sharing a thought.

Reply:
I will revert. thanks. no big preference.
Just trying to reduce the number of redirections :)